python/GafferSceneUITest/SceneViewTest.py

```diff
@@ -372,11 +372,15 @@ def assertLookThroughCamera() :
 				script["camera"]["clippingPlanes"].getValue(),
 				view.viewportGadget().getCamera().parameters()["clippingPlanes"].value
 			)
-			self.assertEqual(
+			self.assertAlmostEqual(
 				script["camera"]["fieldOfView"].getValue(),
-				view.viewportGadget().getCamera().parameters()["projection:fov"].value
+				view.viewportGadget().getCamera().calculateFieldOfView()[0], places = 5
 			)
 
+		# Quick hack - in order to compare FOV, we don't want the 40 border pixels added to the viewport
```

Comment:
Is this necessary because we're now going through an fov->focalLength->fovIncludingTheBorder transformation? Is FOV the best way of presenting the camera setting now, or would focalLength be better anyway? I wonder what Murray would prefer?

Reply:
What I was trying to figure out is why this wasn't necessary before - what changed exactly?

Reply:
Previously, ViewportGadget hacked up the screenWindow, and the test didn't pay any attention to the impact of screenWindow on the actual field of view.  The test just checked that the input fieldOfView parameter matched.

Now I check that the actual rendering frustum matches - but that means that I can actually see the impact of the expansion that ViewportGadget does.

Reply:
> Previously, ViewportGadget hacked up the screenWindow...now I check that the actual rendering frustum matches - but that means that I can actually see the impact of the expansion that ViewportGadget does.

Thanks for the explanation...

> If using the large resolution in the test to get around this offends you, I could add some math with tan/atan to assertLookThroughCamera to explicitly compensate for the border.

No, this'll do for now...